frontend/app/updateRollButton.js

```diff
@@ -11,9 +11,14 @@ import * as getCookie from './checkUserIDCookie';
 export function checkRollValuesEqual(req) {
     // Enables the roll button for the player if the previous
     // roll values are equal
-    const state = JSON.parse(req.responseText);
+    const response = JSON.parse(req.responseText);
+    const roll = response.player_id;
```

Comment:
Should this not be?
```javascript
const roll = response.your_rolls;
```
Unless it's going to be changed but at the moment player_roll_dice() returns as 
```python
{"your_rolls": [4, 1]}
```